modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -88,6 +88,13 @@ def local_bgs_service
         )
       end
 
+      def tracked_item_service
+        @tracked_item_service ||= TrackedItemService.new(
```

Comment:
why the choice to just do `TrackedItemService` instead of `ClaimsApi::TrackedItemService`? Seems to work the same, just asking.

Reply:
Oversight